src/js/controllers/walletHome.js

```diff
@@ -1225,45 +1225,59 @@ angular.module('copayApp.controllers').controller('walletHomeController', functi
     return actions.hasOwnProperty('create');
   };
 
-  this._doSendAll = function(amount) {
+  this._doSendMax = function(amount) {
     this.setForm(null, amount, null);
   };
 
-  this.sendAll = function(totalBytesToSendMax, availableBalanceSat) {
+  this.sendMax = function() {
     var self = this;
-    var availableMaxBalance;
-    var feeToSendMaxStr;
+    var fc = profileService.focusedClient;
     this.error = null;
     this.setOngoingProcess(gettextCatalog.getString('Calculating fee'));
 
     feeService.getCurrentFeeValue(function(err, feePerKb) {
       self.setOngoingProcess();
+
       if (err || lodash.isNull(feePerKb)) {
```

Comment:
it's safer to check for `!_lodash.isNumber(feePerKb)` as this will also take care of `undefined`